src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/PopulateUpdateSubmission.java

```diff
@@ -29,6 +29,7 @@ public class PopulateUpdateSubmission {
      */
     public void populate(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
             UpdateSubmission updateSubmission) {
+        updateSubmission.setEntityNumber(overseasEntitySubmissionDto.getEntityNumber());
```

Comment:
Good idea moving this, can you update the tests just to confirm this is being set?

Reply:
I believe its already tested in testFilingGenerationWhenSuccessfulWithoutTrustsAndWithIdentityChecksForUpdate in FilingServiceTest